client/src/modules/users/UserDepotManagementModal.html

```diff
@@ -19,14 +19,37 @@ <h4><i> {{ UsersDepotModalCtrl.user.display_name }} </i></h4>
     </div>
 
     <span translate> FORM.INFO.USER_DEPOT </span>
+    <div class="checkbox">
+      <label data-root-node>
+        <input type="checkbox" ng-model="UsersDepotModalCtrl.all" ng-change="UsersDepotModalCtrl.setAllNodeValue(UsersDepotModalCtrl.depotsData, UsersDepotModalCtrl.all)" />
+        <strong class="text-capitalize" translate>FORM.LABELS.CHECK_ALL</strong>
+      </label>
+    </div>
+    
+    <div class="panel panel-default" style="margin-bottom : 0px;">
+      <div class="panel-heading clearfix" style="text-align : left">
```

Comment:
Do you need the `clearfix` class here?

Reply:
Also, instead of using both class and style, you can also use the class `text-left`.